src/hooks/aelin/useNftUserAllocation.tsx

```diff
@@ -3,18 +3,17 @@ import { useMemo } from 'react'
 import { BigNumber } from '@ethersproject/bignumber'
 
 import { ParsedAelinPool } from './useAelinPool'
-import { BASE_DECIMALS, ZERO_BN } from '@/src/constants/misc'
+import { ZERO_BN } from '@/src/constants/misc'
 import { useNftSelection } from '@/src/providers/nftSelectionProvider'
 import { ParsedNftCollectionRules } from '@/src/utils/aelinPoolUtils'
 import { formatToken } from '@/src/web3/bigNumber'
 
 function useNftUserAllocation(pool: ParsedAelinPool) {
-  const { lastSelectedNfts, selectedNfts } = useNftSelection()
+  const { selectedNfts } = useNftSelection()
 
   const nfts = useMemo(() => {
-    if (Object.values(lastSelectedNfts).length) return lastSelectedNfts
```

Comment:
We should not remove this, otherwise after an NFT selection, changes won't be stored/reflected.
Maybe this was causing other issues but we should see the logic in `nftSelectionProvider`